tests/test_rescue.py

```diff
@@ -10,569 +10,419 @@
-                with self.assertRaises(TypeError):
-                    self.rescue.code_red = piece
-
-    def test_title(self):
-        """
-        Verifies `Rescue.title` behaves as expected
-        """
-
-        with self.subTest(condition="good"):
-            if self.rescue.title:
-                self.rescue.title = None
-                self.assertIsNone(self.rescue.title)
-
-            title = "foobar express"
-            self.rescue.title = title
-            self.assertEqual(self.rescue.title, title)
-
-        with self.subTest(condition="garbage"):
-            garbage = [[], {}, 42, 22., 0, uuid4()]
-            for piece in garbage:
-                with self.subTest(piece=piece):
-                    with self.assertRaises(TypeError):
-                        self.rescue.title = piece
-
-    def test_rats_good_data(self):
-        """
-        Verfifies the `Rescue.rats` property behaves as expected
-        """
-        self.assertEqual(self.rescue.rats, [])
-        data = [Rats(uuid4(), "unit_Test"), Rats(uuid4(), "icarus")]
-        self.rescue.rats = data
-        self.assertEqual(self.rescue.rats, data)
-
-    def test_rats_bad_types(self):
-        """
-        Verifies the proper exception is raised when `Rescue.rats` is given garbage params
-        """
-        garbage = [{}, -1, 0, .2, (2, 3), uuid4()]
-        for piece in garbage:
-            with self.subTest(piece=piece):
-                with self.assertRaises(TypeError):
-                    self.rescue.rats = piece
-
-    def test_eq_true_branch(self):
-        """
-        Verifies Rescue.__eq__ functions as expected when comparing two rescues
-         - verifies the true branch
-        """
-        # is this total overkill?
-        self.assertTrue(self.rescue == self.rescue)
-
-
-
-    def test_eq_false_branch(self):
-        """
-        Verifies Rescue.__eq__ functions as expected when comparing two rescues
-             - verifies the false branch
-        """
-        foo = Rescue(uuid4(), "snafu", "firestone", "snafu")
-        self.assertFalse(self.rescue == foo)
-
-    def test_platform_valid(self):
-        """
-        Verfiies Rescue.platform setter and getter function as expected when given valid data
-        """
-        platforms = [Platforms.PC, Platforms.PS, Platforms.XB, Platforms.DEFAULT]
-        for platform in platforms:
-            with self.subTest(platform=platform):
-                self.rescue.platform = platform
-                self.assertEqual(self.rescue.platform, platform)
-                self.assertEqual(self.rescue._platform, self.rescue.platform)
-
-    def test_platform_invalid(self):
-        garbage = ["pc", None, 42, 12.2, []]
-        for piece in garbage:
-            with self.subTest(piece=piece):
-                with self.assertRaises(TypeError):
-                    self.rescue.platform = piece
+from Modules.rat_rescue import Rescue
+from utils.ratlib import Status
 
 
 class TestRescuePyTests(object):
```

Comment:
if you are going to move the tests out of the class, you may as well remove the class entirely

Reply:
I'll get .open, and .title covered, as well as fix the indents.  Expect another commit to this PR on Saturday.